sagemaker_neo_compilation_jobs/pytorch_torchvision/code/resnet18.py

```diff
@@ -0,0 +1,79 @@
+import os
+import logging
+import torch
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+def transform_fn(model, payload, request_content_type,
+                 response_content_type):
+    from PIL import Image   # Training container doesn't have this package
```

Comment:
Looks inefficient to import these for every inference. Should we move all the imports to the top?